src/post/post.service.ts

```diff
@@ -582,23 +591,23 @@ export class PostService {
 
         // If the provided handle is invalid, return early
         if (!isHandle(handle)) {
-            throw new Error('Invalid handle');
+            return error('invalid-handle');
         }
 
         if (!defaultAccount || !defaultAccount.id) {
```

Comment:
I think we can simplify this by passing the `accountId` and `accountApId` in for now? Because we know that in this flow this branch never happens

This can become simpler when the migration of this to the View pattern is done